src/graphql/schema/label.ts

```diff
@@ -4,6 +4,7 @@ export default gql`
   extend type Query {
     numLabels: Int!
     getLabels: [Label!]!
+    getAreaLabels(idfront: String!): [Label]!
```

Comment:
```suggestion
    getAreaLabels(idfront: String!): [Label!]!
```
We never want to have a null value in the array.